### PR TITLE
release/0.1.0.beta5

### PR DESCRIPTION
**Pull request by ankitmishra262**
## 🗒️ Release notes

Take a look at our [CONTRIBUTING.md](TBD) guidelines for information on how to complete this section.

### User-facing change?
- [ ] No. Skip the rest of this section.
- [x] Yes. Complete the sections below to define what is included in the release notes for super.AI users.

## Documentation
*Provide a link to the documentation of the new feature or paste an example on how to use it inline*
- [ ] Link:

### How should the PR be classified in the release notes? Choose one:

- [x] `Major new feature` - A new landmark feature that notably expands the product 
- [ ] `New feature` - Any new functionality that is added
- [x] `Bugfix` - Anything that corrects an error or unexpected behavior
- [x] `Enhancement` - An improvement/change that was not a bugfix

### Short description of the change and how it benefits the user

SuperAI SDK Release (0.1.0.beta5)

New Features:
- AI Training container creation, starting the training process from the SDK, loading local datasets
- Addition of post-process endpoint to DP server
- Starting, retrieving and deleting training
- Handling root id child versions
- Workflow deletion rail guards
- Support for weight loading and tar files

Fixes:
- Enabled CLI command failures in the SDK
- Adding compatibility of 3-hero methods for the new schema
- Improved training for the new schema
- Remove image age check
- Credential and region specification improvements for CI
- Installing the latest release of superai-SDK in AI base images and conda environments

### Long description for major new features

(2+ sentences explaining what the feature is, why it's important, and how it benefits the user)

### CODE REVIEW

File: Earthfile [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+    BUILD +semgrep
+    BUILD +pre-commit
+
+PIP_INSTALL:
+    COMMAND
+    ARG REQTARGET="."
+    IF  [ "$INTERNAL" = "true" ]
+        RUN --mount=type=secret,id=+secrets/aws,target=/root/.aws/credentials \
+            --mount=type=cache,target=/root/.cache/pip \
+            aws codeartifact login --tool pip --domain superai --repository pypi-superai-internal && \
```
Comment (engarpe) [inline Earthfile:39]: We shouldn't require the superai-internal repository, we should use the pypi-superai

File: docs/examples/ai/resources/runDir/run_this.sh [hunk 2]
```diff
@@ -1,3 +1,6 @@
 aws codeartifact login --tool pip --domain superai --repository pypi-superai-internal
```
Comment (engarpe) [inline docs/examples/ai/resources/runDir/run_this.sh:1]: Won't work with the aws credentials that you get with `superai login`

**Review by engarpe — CHANGES_REQUESTED** [review]

File: Earthfile (hunk 1 above)
Comment (ankitmishra262) [reply Earthfile:39]: I checked the idea behind this. This is handled internally in this file here https://github.com/mysuperai/superai-sdk-automation/blob/main/vars/buildPipeline.groovy. The logic differs for internal and external SDK

File: docs/examples/ai/resources/runDir/run_this.sh (hunk 2 above)
Comment (ankitmishra262) [reply docs/examples/ai/resources/runDir/run_this.sh:1]: Will change this

File: docs/examples/ai/resources/runDir/run_this.sh [hunk 3]
```diff
@@ -1,4 +1,3 @@
-aws codeartifact login --tool pip --domain superai --repository pypi-superai-internal
```
Comment (engarpe) [inline docs/examples/ai/resources/runDir/run_this.sh:1]: I guess you need to still do `aws codeartifact login --tool pip --domain superai --repository pypi-superai` externals have access to this

File: Earthfile (hunk 1 above)
Comment (engarpe) [reply Earthfile:39]: If you do `aws codeartifact login --tool pip --domain superai --repository pypi-superai` should work for externals. But please test it first

**Review by engarpe — CHANGES_REQUESTED** [review]

File: docs/examples/ai/resources/runDir/run_this.sh (hunk 3 above)
Comment (ankitmishra262) [reply docs/examples/ai/resources/runDir/run_this.sh:1]: That is done internally in the s2i script. So it's not required to explicitly specify the codeartifact login here.

**Review by engarpe — APPROVED** [review]

**Pull request merged**